src/fides/api/service/connectors/saas/connector_registry_service.py

```diff
@@ -339,6 +341,10 @@ def create_connection_config_from_template_no_save(
     if template_values.name:
         data["name"] = template_values.name
 
+    enabled_actions = getattr(template_values, "enabled_actions", None)
+    if enabled_actions is not None:
+        data["enabled_actions"] = enabled_actions
```

Comment:
This function is used by both fides and fidesplus endpoints so we need to perform this attribute check to map the `enabled_actions` attribute which is only found in fidesplus.

Reply:
yeah, this is an interesting pattern. i can't say i _love_ it, it feels like a development footgun moving forward - we're implying (in a pretty cryptic manner) that someone developing here in fides knows about something defined over in fidesplus. i'm wouldn't be surprised if we do that in some other places already, but my gut-instinct tells me it's something we should try to avoid if possible. 

i'll make a broader comment on the PR at this point, since i think this is a holistic concern - not tied to this particular code block, even though this is where it's manifested